src/chainstate/stacks/boot/pox-2.clar

```diff
@@ -720,9 +770,18 @@
           lock-period: lock-period })
 
       ;; return the lock-up information, so the node can actually carry out the lock. 
-      (ok { stacker: stacker,
-            lock-amount: amount-ustx,
-            unlock-burn-height: unlock-burn-height })))
+      (ok (merge event-info {
+            name: "delegate-stack-stx", 
+            data: {
+                lock-amount: amount-ustx,
+                unlock-burn-height: unlock-burn-height,
```

Comment:
@pavitthrap can you confirm that the `unlock-burn-height` specified here and in a few others will always match the value of `burnchain-unlock-height` from `event-info` data in the same payload?